### PR TITLE
Add fetch step for wa/state. Fixes #479

### LINKED ISSUE #479: fetch wa/state
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://vaccinelocator.doh.wa.gov/

Put your script in a file named: `wa/state/fetch.sh` (or `fetch.sh`, etc.)

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch wa/state
    ```

### PR DESCRIPTION
**Pull request by Satshabad**
# Fetch State for WA

| Key Details |
|-|
| Resolves #479 |
State: WA |
Site: state |

## Notes
Very convenient graphql API with self documented interface. Seemed fine to grab it all in one page. About 2k results when I tested it out.

## Data sample

```json
{"data":{"searchLocations":{"paging":{"total":1921,"pageNum":1,"pageSize":1000000},"locations":[{"locationId":"recT5VYLb0qwyVTSz","locationName":"North Basin - Davenport","locationType":null,"providerId":null,"providerName":null,"departmentId":null,"departmentName":null,"addressLine1":"100 3rd Street","addressLine2":null,"city":"Davenport","state":"Washington","county":"Lincoln","zipcode":"99122","latitude":47.65713,"longitude":-118.14546,"description":null,"contactFirstName":null,"contactLastName":null,"fax":null,"phone":"(509) 725-7501","email":null,"schedulingLink":"https://www.lincolnhospital.org/health-resources/covid-19-update/","vaccineAvailability":"AVAILABLE","infoLink":null,"timeZoneId":null,"directions":"We are currently scheduling weekly vaccine clinics based on the number of vaccines we receive and individual eligibility. The clinics are by appointment only.\n\nPlease call 509.725.7501 and ask to be added to our wait-list. Please note that the vaccine is still available in limited quantities and our demand is higher than our current supply. We will do the best we can to get your scheduled as quickly as possible.","updatedAt":"2021-05-08T17:58:06.647Z","rawDataSourceName":"CovidWaFn","vaccineTypes":[],"accessibleParking":null,"additionalSupports":null,"commCardAvailable":null,"commCardBrailleAvailable":null,"driveupSite":null,"interpretersAvailable":null,"interpretersDesc":null,"supportUrl":null,"waitingArea":null,"walkupSite":null,"wheelchairAccessible":null,"scheduleOnline":null,"scheduleByPhone":null,"scheduleByEmail":null,"walkIn":null,"waitList":null},{"locationId":"recmLmhezm8iIK6t0","locationName":"Valley View Health Center - Winlock","locationType":null,"providerId":null,"providerName":null,"departmentId":nu ...
```

## Before Opening a PR
- [x] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [x] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

File: vaccine_feed_ingest/runners/wa/state/fetch.py
Comment (bryanculbertson) [inline vaccine_feed_ingest/runners/wa/state/fetch.py:20]: I am slightly concerned that they will eventually add a max `pageSize` and this will break. I am going to approve and merge, but if you have the time to go back and implement paging that would make this more robust.

**Review by bryanculbertson — APPROVED** [review]

**Comment by bryanculbertson** [comment]
@Satshabad SuperLinter is being too string and claiming the indenting is wrong for the GraphQL string :-/

Best fix I can think of is to exclude the block from the linter:
```
// editorconfig-checker-disable
const myTemplateString = `
  first line 
     wrongly indended line because it needs to be
`
// editorconfig-checker-enable
```

https://github.com/editorconfig-checker/editorconfig-checker#excluding-blocks

File: vaccine_feed_ingest/runners/wa/state/fetch.py
Comment (Satshabad) [reply vaccine_feed_ingest/runners/wa/state/fetch.py:20]: I could definitely do that. What do we think is a safe page size though 100? 50? I'm also quite available and could quickly fix this if they implemented one.

Comment (bryanculbertson) [reply vaccine_feed_ingest/runners/wa/state/fetch.py:21]: I think a safe page size would be `1000`, but we can also hold off on implementing paging until it becomes important.

**Review by bryanculbertson — APPROVED** [review]

**Pull request merged**
